spec.bs

```diff
@@ -2495,6 +2495,23 @@ container/fenced navigable=] will fail, as outlined in [[#navigation-patch]].
 
 </div>
 
+<h4 id=allow-automatic-beacons>The
+\`<a http-header><code>Allow-Fenced-Frame-Automatic-Beacons</code></a>\` HTTP response header</h4>
+
+A {{Document}} that is not [=same origin=] with its [=fenced frame config instance=]'s [=fenced
+frame config instance/mapped url=] can opt into sending an automatic beacon when it navigates by
```

Comment:
Remind me how a fenced frame config with a mapped URL can load a document whose origin is different from that mapped URL? Redirects?

Reply:
If the document navigates itself to a new URL, the fenced frame config instance [persists across the cross-document navigation](https://wicg.github.io/fenced-frame/#creating-browsing-contexts-patch:~:text=Inner%2Dcontent%2Dinitiated%20navigations%3A%20The%20fenced%20frame%20config%20instance%20referenced%20in%20the%20immediately%2Dpreceding%20point%20persists%20through%20this%20kind%20of%20cross%2DDocument%20navigation.).

Cross-origin child iframes will also [get the fenced frame config instance of its fenced traversable navigable](https://wicg.github.io/fenced-frame/#:~:text=(2)%20inherited%20during%20initial%20Document%20creation%20for%20Documents%20in%20child%20navigables%20whose%20traversable%20navigable%20is%20a%20fenced%20navigable%2C%20as%20handled%20in%20the%20%C2%A7%E2%80%AF3.3%20Modifications%20to%20creating%20browsing%20contexts%20section.), which will result in the origins being different.

Reply:
OK thanks